packages/web-components/fast-components/src/accordion-item/index.ts

```diff
@@ -1,28 +1,55 @@
-import { customElement } from "@microsoft/fast-element";
 import {
     AccordionItem,
-    AccordionItemTemplate as template,
+    AccordionItemOptions,
+    accordionItemTemplate as template,
 } from "@microsoft/fast-foundation";
-import { AccordionItemStyles as styles } from "./accordion-item.styles";
+import { accordionItemStyles as styles } from "./accordion-item.styles";
 
 /**
- * The FAST Accordion Item Element. Implements {@link @microsoft/fast-foundation#AccordionItem},
- * {@link @microsoft/fast-foundation#AccordionItemTemplate}
+ * A function that returns a {@link @microsoft/fast-foundation#AccordionItem} registration for configuring the component with a DesignSystem.
+ * Implements {@link @microsoft/fast-foundation#accordionItemTemplate}
  *
  *
  * @public
  * @remarks
- * HTML Element: \<fast-accordion-item\>
+ * Generates HTML Element: \<fast-accordion-item\>
  */
-@customElement({
-    name: "fast-accordion-item",
+export const fastAccordionItem = AccordionItem.compose<AccordionItemOptions>({
```

Comment:
Oh one thing I noticed for all components out of `fast-components`: we need to export the component class the creates the definition, otherwise tags cannot be retrieved from `context.tagFor()` because that method requires the class ctor as an argument.

Reply:
@nicholasrice can we put a PR together for that (and maybe the deps) so nothing is missed?

Reply:
DONE!